tests/subsys/tracing/tracing_sys/testcase.yaml

```
@@ -0,0 +1,8 @@
+tests:
+  tracing.disable:
```

I suppose the rationale behind having this test under `subsys/tracing/tracing_sys/` rather than just `subsys/tracing/` is that you expect more tracing-related tests to be added in the future.

In that case, the testcases in this test should be named to support that.
```suggestion
  tracing.tracing_sys.disable:
```

---

Yes, and I have corrected the name, thanks.